README.md

```diff
@@ -551,6 +560,33 @@ strategies:
           - {key: environment, operator: NotIn, values: [dev]}
 ```
 
+
+### Node Fit filtering
+
+All strategies are able to configure a `nodeFit` boolean parameter. If set to `true` the descheduler will consider whether or not the pods that meet eviction criteria will fit on other nodes before evicting them. If a pod cannot be rescheduled to another node, it will not be evicted. Currently the following criteria are considered when setting `nodeFit` to `true`:
+- A `nodeSelector` on the pod
+- Any `Tolerations` on the pod and any `Taints` on the other nodes
+- `nodeAffinity` on the pod
+- Whether any of the other nodes are marked as `unschedulable`
+
+E.g.
+
+```yaml
+apiVersion: "descheduler/v1alpha1"
+kind: "DeschedulerPolicy"
+strategies:
+  "PodLifeTime":
+     enabled: true
+     params:
+        podLifeTime:
+          maxPodLifeTimeSeconds: 86400
+        nodeFit: true
+```
+
+Note that node fit filtering references the current pod spec, and not that of it's owner. Thus, if the pod is owned by a ReplicationController (and that ReplicationController was modified recently), the pod may be running with an outdated spec, which the descheduler will reference when determining node fit. This is expected behavior as the descheduler is a "best-effort" mechanism. 
```

Comment:
The pod can be also owned by a ReplicaSet or StatefulSet. What about `Thus, if the pod is managed by a controller which does not recreate all replicas everytime a pod template is updated, ...` ?

Reply:
Added, thanks.